mne/io/meas_info.py

```diff
@@ -2228,17 +2228,16 @@ def anonymize_info(info, daysback=None, keep_his=False, verbose=None):
     none_meas_date = info['meas_date'] is None
 
     if none_meas_date:
-        warn('Input info has \'meas_date\' set to None.'
-             ' Removing all information from time/date structures.'
-             ' *NOT* performing any time shifts')
-        info['meas_date'] = None
```

Comment:
This line was not necessary?

Reply:
Yes, I believe it's a cruft leftover from a previous refactoring.

The code goes like this:

```python
    none_meas_date = info['meas_date'] is None

    if none_meas_date:
```
so when we're inside the `if` branch, `info['meas_date']` is already `None`